insights/specs/datasources/httpd.py

```diff
@@ -78,7 +78,7 @@ def _get_all_include_conf(root, glob_path):
                             section_number = section_number - 1
                         elif line.startswith("<"):
                             section_number = section_number + 1
-                        if section_number == 0 and (line.startswith("Include ") or line.startswith("IncludeOptional ")):
+                        if section_number == 0 and (line.lower().startswith("include ") or line.lower().startswith("includeoptional ")):
```

Comment:
@wushiqinlou As this line also mainly check for `line.startswith()`, what about change this `if` to `elif`?

Reply:
@JoySnow - updated, please check